artic/README.md

```diff
@@ -7,43 +7,92 @@ title: Artic
+
+   If within-sample variation affects a primer binding site, the primer is expected to bind preferentially to the perfectly matching reference sequence as opposed to sequences carrying the variant allele, i.e., amplicons generated with such biased primers will more likely capture the sequence of viral template molecules without the variant than with it, and this extends to any linked variants within the amplicon region.
+
+   In order to avoid an allele frequency bias for such linked variants, one solution is to not use data from biased
+   amplicons (for which at least one primer binding site displays within-sample variation) at all.
+
+   We implement this idea and perform a first round of variant calling to detect primer binding site variants. We then use *iVar* again to remove reads from biased amplicons prior to a second round of *unbiased* variant calling.
+
+   Since removing the information from whole amplicons is a rather drastic measure, which will leave a gap of no coverage in the tiled layout of reads, we then compare the variant lists before and after amplicon removal, and add variants lost in the second round of calling back in to the final list of variants. Those variants get flagged as not having passed amplicon removal though to indicate that their call statistics might not be reliable.
 
 | ![](./img/wf.png) |
 |:------------------:|
-| Galaxy workflow for ARTIC analysis of amplicon data from Illumina sequencing |
+| Galaxy workflow for the analysis of Illumina paired-end sequenced ARTIC amplicon data |
+
+### Required Inputs
+
+The workflow requires:
+
+1. A collection of the forward and reverse reads in `fastq` format of all samples to be analyzed
 
-The workflow makes use of the [iVar software](https://andersen-lab.github.io/ivar/html/) from the Andersen Lab at the
-Scripps Research Institute in California for trimming the primer section of reads and also, currently, for variant calling.
-The ARTIC primer BED files for use with this workflow can be found at the
-[ARTIC nCov2019 Github](https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019). The correct primer BED file for the version of the primers used during sequencing should be used.
+2. A BED file describing the binding sites of all primers used to generate the tiled amplicons
 
-Note that the V3 BED file has a [known](https://github.com/artic-network/artic-ncov2019/issues/12) formatting
-problem which can be solved with a command line command (`awk 'BEGIN {OFS="\t"; FS="\t"} $5=60' ARTICv3.bed  >ARTICv3_fixed.bed`)
-or the equivalent awk program (`BEGIN {OFS="\t"; FS="\t"} $5=60}`) run in the Galaxy awk tool.
+   The ARTIC project provides BED files for use with this workflow as part of their [SARS-CoV-2 resources repo](https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019).
+   
+   For reproducibility we provide here copies of the:
+   
+   - [v3 primer scheme BED file](nCoV-2019_v1.bed)
```

Comment:
The v3 and v1 links point to the same file

Reply:
Tahnks for catching @Slugger70!